src/App.js

```diff
@@ -1,5 +1,43 @@
-function App() {
-  return <div>Hello world</div>
-}
+import { useState } from 'react'
+import { v4 as uuid } from 'uuid'
```

Comment:
Shouldn't it be " import { v4 as uuidv4 } from 'uuid' "

Reply:
I guess it's up to you. You might call is 'currywurst'.

Reply:
You're right Nina but I like it shorter :)